File: BankAccount.java

public class BankAccount {

 private string m_customerName;

 public string getM_customerName() {
  return m_customerName;
 }

 public void setM_customerName(string m_customerName) {
  this.m_customerName = m_customerName;
 }
 private double m_balance;

 public double getM_balance() {
  return m_balance;
 }

 public void setM_balance(double m_balance) {
  this.m_balance = m_balance;
 }
public  bankAccount() {
}

public  bankAccount(string customerName, double balance) {
}

public  bankAccount() {
   m_customerName = customerName;
   balance = balance;
}

public  bankAccount(string customerName, double balance) {
   m_customerName = customerName;
   balance = balance;
}

public  debit(double amount) {
m_balance-=amount
}

public  credit(double amount) {
m_balance+=amount
}

public void main() {
   double vCredit;
   double vDebit;
   vCredit = 5.77;
   vDebit = 11.22;
   BankAccount ba = new BankAccount();
   ba.setM_customerName(m_customerName);
   ba.setM_balance(m_balance);
   ba.credit(vCredit);
   ba.debit(vDebit);
   Console.WriteLine(m_balance);
}


}

File: BankAccount.swift

class BankAccount {

 private var m_customerName: String = ""


 public func  getM_customerName()  -> String { 
  return self.m_customerName
 }

 public func setM_customerName(m_customerName: String) {
  self.m_customerName = m_customerName
 }
 private var m_balance: Double = 0


 public func  getM_balance()  -> Double { 
  return self.m_balance
 }

 public func setM_balance(m_balance: Double) {
  self.m_balance = m_balance
 }
func  bankAccount()  {
}
func  bankAccount(customerName: String, balance: Double)  {
}
func  bankAccount()  {
   m_customerName = customerName
   balance = balance
}
func  bankAccount(customerName: String, balance: Double)  {
   m_customerName = customerName
   balance = balance
}
func  debit(amount: Double)  {
m_balance-=amount
}
func  credit(amount: Double)  {
m_balance+=amount
}
func  main()  -> Void {
   var vCredit: Double	     
   var vDebit: Double	     
   vCredit = 5.77
   vDebit = 11.22
   let ba =  BankAccount()	     
   ba.setM_customerName(m_customerName: m_customerName)	  
   ba.setM_balance(m_balance: m_balance)	  
   ba.credit(vCredit: vCredit)	  
   ba.debit(vDebit: vDebit)	  
   Console.WriteLine(m_balance: m_balance)	  
}

}
